src/comments/CommentRepository.kt

```diff
@@ -0,0 +1,45 @@
+package com.sniper.ktor.comments
+
+import com.sniper.ktor.EntityNotFoundError
+import java.util.concurrent.CopyOnWriteArraySet
+import java.util.concurrent.atomic.AtomicInteger
+
+object CommentRepository {
```

Comment:
Maybe we shall need generic repository class , because we repeat  the implementation

Reply:
Indeed, I will do it in the next PR